napari/components/add_layers_mixin.py

```diff
@@ -237,7 +237,9 @@ def add_image(
             # so that we can use {k: next(v) for k, v in kwargs.items()} below
             for key, val in kwargs.items():
                 if key == 'colormap' and val is None:
-                    if n_channels < 3:
+                    if n_channels == 1:
```

Comment:
This is more of a curiosity and if we do want to change this, it's definitely out of the scope of this PR, but why do we do these types of kwarg checks/cleaning in `add_image()` rather than the `Image` model init method? It seems like this would lead to a divergence in how the model behaves on its own versus how it effectively behaves with the viewer.

Reply:
Because `Image` doesn't have `channel_axis`, since that requires the creation of multiple Image layers

Reply:
Oh, right 🤦 Thanks for the explanation!